docs/releases.md

```diff
@@ -28,8 +28,11 @@ Both the `argo-server` and `argocli` should be the same version as the controlle
 
 ## Release Cycle
 
-New minor versions are released roughly every 6 months. Release candidates for each major release are typically available
-for 4-6 weeks before the release becomes generally available.
+New minor versions are released roughly every 6 months.
```

Comment:
we're not going to try do quarterly?

Reply:
Also, I assume that if there's backwards incompatibility then we alternatively release a major release but at the same cadence?

Reply:
@terrytangyuan what is the cadence you want to advertise? 6 months or 3, or maybe 3-6?

Reply:
lower expectation should be good. I don't think we have resources to do quarterly

Reply:
As Caelan mentioned, it was adjusted in #11239. This PR does not change that line

Reply:
> As Caelan mentioned, it was adjusted in #11239. This PR does not change that line

Sure

Reply:
@tico24 So, when I wrote above "Also, I assume that if there's backwards incompatibility then we alternatively release a major release but at the same cadence?" - I was thinking that maybe we should change the sentence "New minor versions are released roughly every 6 months"? It doesn't say anything about major releases.